src/components/flame-graph/FlameGraph.js

```diff
@@ -222,9 +221,13 @@ class FlameGraphImpl extends React.PureComponent<Props> {
       rightClickedCallNodeIndex,
       changeSelectedCallNode,
       handleCallNodeTransformShortcut,
-      openSourceView,
     } = this.props;
 
+    if (event.key === 'Enter') {
+      this._onCallNodeEnterOrDoubleClick(selectedCallNodeIndex);
+      return;
+    }
+
```

Comment:
Is there a reason you duplicated this code here?

I was under the impression that this would regress the "right click, use the context menu with the keyboard, then press enter" case (because this uses `selectedCallNodeIndex` and not `rightClickedCallNodeIndex`), but after trying it looks like this doesn't regress this case and I'm no sure why. (note: we also have issue #4435 when navigating the flame graph context menu with the keyboard.)

Well I guess the question is: is there a reason this is here in addition to below, or is this just an oversight?

Reply:
Oh this was an oversight. I noticed it during rebasing but then forgot to fix it.